instrumentation/github.com/aws/aws-lambda-go/otellambda/README.md

```diff
@@ -16,38 +16,65 @@ See [./example](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/
 
 ## Usage
 
-Create a sample Lambda Go application such as below.
+Create a sample Lambda Go application instrumented by the `otellambda` package such as below.
 
 ```go
 package main
 
 import (
 	"context"
 	"fmt"
+	
 	"github.com/aws/aws-lambda-go/lambda"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda"
 )
 
-type MyEvent struct {
-	Name string `json:"name"`
-}
-
-func HandleRequest(ctx context.Context, name MyEvent) (string, error) {
-	return fmt.Sprintf("Hello %s!", name.Name ), nil
+func HandleRequest(ctx context.Context) (error) {
+	fmt.Println("Hello World!" )
+	return nil
 }
 
 func main() {
-	lambda.Start(HandleRequest)
+	ctx := context.Background()
+	lambda.Start(otellambda.InstrumentHandler(HandleRequest(ctx)))
 }
 ```
 
-Now use the provided wrapper to instrument your basic Lambda function:
+Now configure the instrumentation with the provided options to export traces to AWS X-Ray via [the OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) running as a Lambda Extension. Instructions for running the OTel Collector as a Lambda Extension can be found in the [AWS OpenTelemetry Documentation](https://aws-otel.github.io/docs/getting-started/lambda).
```

Comment:
Do not conflate the `otellambda` instrumentation and the `xrayconfig` setup packages.  This is documentation for `otellambda` and should not need to mention X-Ray at all.